ogc/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: OGC protocols and formats
+layout: intro
```

Comment:
The site doesn't have an layout named "intro", you could used "default", or we could add one.